src/main/java/org/jabref/gui/exporter/CreateModifyExporterDialogView.java

```diff
@@ -0,0 +1,73 @@
+package org.jabref.gui.exporter;
+
+import java.util.Optional;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.TextField;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.logic.journals.JournalAbbreviationLoader;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.preferences.PreferencesService;
+
+import com.airhacks.afterburner.views.ViewLoader;
+
+public class CreateModifyExporterDialogView extends BaseDialog<Optional<ExporterViewModel>> {
```

Comment:
This should be `BaseDialog<ExporterViewModel>` as the `show` method already returns an `Optional` (which is empty if the user choose to close the dialog using e.g. the cancel button). Then, return `null` in `setResultConverter` when the user canceled the action or the exporter is invalid.

Reply:
I just pushed a fix for this.